---
layout: page
title: CBB752 Spring 2017
tagline: Final Project
---

Project 1.3: Analyzing the Protein Coding Mutations in the Zimmerome
------------------


Table of Contents
-----------------------




**Contributors**
 -Writing: Ramya
 -Coding: Nir
 -Pipeline: Megan

### Introduction: What is Variant Prioritization?
Since the advent of rapid and cheap whole genome sequencing technologies, we have amassed a large diverse collection of human genomes. Interpreting this large amount of data is still challenging, and one such challenge is understanding the function of single nucleotide variations (SNVs) that often arise in individual genomes compared to reference sets which also include many single nucleotide polymorphisms (SNPs). Specifically, identifying pathogenic or deleterious mutations compared to common background mutations is a key challenge. Often, deleterious mutations are non-synonymous, meaning that they result in the change of one amino acid to another. The average individual has over 10 million SNPs in the whole genome, about 20,000 of which are in the exome or the protein-coding portion of the genome. Of course, not all of these are deleterious - the number of potentially deleterious mutations in the exome often amounts to about several hundred (1)(2). This problem of identifying relevant disease-causing mutations or variants compared to common mutations is called variant prioritization.
Here, we aim to prioritize variants in the exome of subjectZ. SubjectZ has 22,981 exome SNVs of which 1,800 are rare SNVs (4). Furthermore, of these rare SNVs, 1,018 were found to be nonsynonymous. Both a denovo approach and existing tools such as SIFT, PolyPhen, and Provean. The methodology used to prioritize SNVs in the denovo tool will be explained and compared to those used in other approaches. The results of our tool and those from the mentioned tools are detailed in the Pipeline and Coding sections of this text.





### Writing:








### Coding:


#### Documentation:


#### Results:







### Pipeline:


#### Documentation:


#### Results:









#### Conclusions:








#### References:

 References can be included here or at the end of each relevant section.
 
 
